tools/Microsoft.StandardsPolice/project.json

```diff
@@ -1,6 +1,10 @@
 ﻿{
   "version": "1.0.0-*",
 
+  "compilationOptions": {
```

Comment:
It's a build-time dependency, not a real dependency, so it shouldn't matter. Did you have any issues without it?